math/src/test/kotlin/ktx/math/rangesTest.kt

```diff
@@ -232,4 +235,23 @@ class RangesTest {
     }
     assert(values.all { it in range })
   }
+
+  @Test
+  fun `should lerp`() {
+    val start = 150f
+    val end = 300f
+    val progress = 0.75f
+    val range = start..end
+    assert(range.lerp(progress) in 262.5f amid 0.000001f)
+  }
+
+  @Test
+  fun `should interpolate`() {
+    val start = 0f
+    val end = 1f
+    val progress = 0.5f
+    val range = start..end
+    println(range.interpolate(progress, Interpolation.sineOut))
+    assert(range.interpolate(progress, Interpolation.sineOut) in 0.7071f amid 0.001f)
```

Comment:
You're using Kotlin's built-in `assert` instead of JUnit assertions. Beside the different error message, afaik this utility might have different behavior depending on the JVM flags:

> Throws an `AssertionError` if the value is `false` and runtime assertions have been enabled on the JVM using the `-ea` JVM option.

It might still work, but I'd prefer if you used `Assert.assertEquals`. I think it also provides a float overload with tolerance, so you won't need `amid`.

Reply:
Same applies to both tests.